tests/test_manageiq_provider.py

```diff
@@ -304,7 +304,7 @@ def test_will_update_amazon_provider_if_present(miq, miq_api_class, amazon_endpo
         "other region")
     assert res_args == {
         'changed': True,
-        'msg': 'Successful update of {} provider. Authentication: All Valid. Refreshing provider inventory'.format(AMAZON_PROVIDER_NAME),
+        'msg': 'Successful update of {} provider. Authentication: {{}}. Refreshing provider inventory'.format(AMAZON_PROVIDER_NAME),
```

Comment:
@cben yes, `authtypes_to_verify` is empty in this case since the amazon provider region is not part of the default endpoint.
If we want to revalidate after region or zone update we need a change in the current logic:
https://github.com/dkorn/manageiq-ansible-module/blob/93bd719b411042c2469843edc06daf5d4acde056/library/manageiq_provider.py#L390